data/coalition/heliarch intro.txt

```diff
@@ -2020,16 +2020,22 @@ mission "Heliarch Drills 3"
 		system "Torbab"
 		ship "Heliarch Punisher (Scrappy)" "Awaiting Adversary"
 		on kill
```

Comment:
Should this perhaps be `on destroy`? If you board and then capture the ship, you'll get both dialogs.

Reply:
I originally had an `on capture` as well with its own dialogue, but like you say it did both conversations, and I took it out and made the one phrase work for either destruction or capture. 

If we want to bring back the capture dialog we can do that, though a capture condition would only have to be the dialog at that point since you're already getting hit with the rep loss and fail for boarding in the first place.

Reply:
Brought it back.